travis-wait-mvn-build.sh

```diff
@@ -6,5 +6,3 @@ function write_visual_bells() {
   done
 }
 write_visual_bells&
-
-set -o pipefail && mvn cobertura:cobertura | grep -v "DEBUG"
```

Comment:
Removed the explicit Cobertura invocation and enabled tests in the main build process to reduce the build overhead.
JaCoCo instruments the tests in the main build process now and also supports tests run in the Maven "integration-test" phase, which would trigger all Maven stages to be executed until at least "integration-test" again after the main build.